src/components/lyrics/index.tsx

```
@@ -1,18 +1,20 @@
 
 import { addToast, useDisclosure } from "@heroui/react";
-import { RiTBoxLine } from "@remixicon/react";
+import { RiTBoxLine, RiEditLine } from "@remixicon/react";
 import clsx from "classnames";
 import { debounce } from "es-toolkit";
 
 import type { WebPlayerParams } from "@/service/web-player";
 
 import { usePlayList } from "@/store/play-list";
 import { usePlayProgress } from "@/store/play-progress";
+import { useFullScreenPlayerSettings } from "@/store/full-screen-player-settings";
```

ESLint

[error] Expected "@/store/full-screen-player-settings" to come before "@/store/play-progress".

(perfectionist/sort-imports)